sandbox/joseph-tableau-connector-2019/json_generator_limit_1000.py

```diff
@@ -0,0 +1,27 @@
+import pymysql
```

Comment:
Since there is only a little different from this file to `json_generator_limit.py`, 
can we merge these two files into one?